doc/classes/CodeEdit.xml

```diff
@@ -181,7 +181,7 @@
 		</theme_item>
 		<theme_item name="font_color_readonly" type="Color" default="Color( 0.88, 0.88, 0.88, 0.5 )">
```

Comment:
This one was likely added later on to match the `font_color_*` construct, but it should likely be renamed too.

Reply:
Updated to also rename:
- `font_color_accel` -> `font_accelerator_color`
- `font_color_bg` -> `font_unselected_color`
- `font_color_disabled` -> `font_disabled_color`
- `font_color_fg` -> `font_selected_color`
- `font_color_hover` -> `font_hover_color`
- `font_color_hover_pressed` -> `font_hover_pressed_color`
- `font_color_pressed` -> `font_pressed_color`
- `font_color_readonly` -> `font_readonly_color`
- `font_color_shadow` -> `font_shadow_color`
- `font_color_uneditable` -> `font_uneditable_color`
- `font_outline_modulate` -> `font_outline_color`
- `icon_color_disabled` -> `icon_disabled_color`
- `icon_color_hover` -> `icon_hover_color`
- `icon_color_hover_pressed` -> `icon_hover_pressed_color`
- `icon_color_normal` -> `icon_normal_color`
- `icon_color_pressed` -> `icon_pressed_color`
- `tab_fg` -> `tab_selected`
- `tab_bg` -> `tab_unselected`

Note 1: I've renamed `font_outline_modulate` -> `font_outline_color`. However, let me know if it should remain `font_outline_modulate`. My thoughts are that although it actually just modulates the outline colour, this name makes more sense and we can easily change the documentation to reflect that.
Note 2: I've changed `fg` to `selected` and `bg` to `unselected`, which apply to tabs, and better reflects what they refer to.